src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResourceTest.kt

```diff
@@ -3,11 +3,11 @@ package uk.gov.justice.digital.hmpps.incentivesapi.resource
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.IepReview
-import uk.gov.justice.digital.hmpps.incentivesapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.incentivesapi.integration.SqsIntegrationTestBase
 import java.time.LocalDate.now
 import java.time.format.DateTimeFormatter
 
-class IepLevelResourceTest : IntegrationTestBase() {
+class IepLevelResourceTest : SqsIntegrationTestBase() {
```

Comment:
Could be worth verifying that `snsService` and `auditService` have been invoked as expected for `addIepReview` functions

Reply:
yes was going to do in next PR